File: 프로그래머스/0/120869. 외계어 사전/외계어 사전.py
def solution(spell, dic):
    cnt = []
    alright = []
    for i in range(len(dic)):
        cnt.append([])
    for i in spell:
        alright.append(1)
        for j in range(len(dic)):
            cnt[j].append(dic[j].count(i))
    if alright in cnt:
        return 1
    else:
        return 2

File: 프로그래머스/0/120869. 외계어 사전/외계어 사전.java
class Solution {
    public int solution(String[] spell, String[] dic) {
        int answer = 2;
        for (String word : dic) {
            for (int i = 0; i < spell.length; i++) {
                if (word.contains(spell[i])) {
                    if (i == spell.length - 1) {
                        answer = 1;
                        break;
                    }
                    continue;
                } else {break;}
            }
            if (answer == 1) {break;}
        }
        return answer;
    }
}